builtin_interfaces/Header header
int32 track_id
float32 x
float32 y
float32 w
float32 h
float32 cx
float32 cy
string room_id
